evmjit/libevmjit/Common.h

```diff
@@ -1,11 +1,12 @@
 #pragma once
 
-#include <vector>
 #include <tuple>
 #include <cstdint>
 
 #ifdef _MSC_VER
-#define EXPORT __declspec(dllexport)
+#define EXPORT __declspec(dllexport)
+#define _ALLOW_KEYWORD_MACROS
+#define noexcept throw()
```

Comment:
Because MSVC 2013 does not support noexcept yet. The workaround is to use `throw()` for now. cpp-ethereum does the same.